src/modules/export/pdf/InterlinearPdfGenerator.ts

```diff
@@ -0,0 +1,445 @@
+import PDFDocument from "pdfkit";
+import { Readable } from "stream";
+import {
+  InterlinearChapterResult,
+  InterlinearVerse,
+} from "../data-access/InterlinearQueryService";
+import path from "path";
+import fs from "fs";
+
+export interface PdfGeneratorOptions {
+  pageSize?: "letter" | "a4";
+  layout: "standard" | "parallel";
+  direction?: "ltr" | "rtl";
+  header?: { title?: string; subtitle?: string };
+  footer?: { generatedAt?: Date; pageOffset?: number; pageTotal?: number };
+  sourceScript?: "hebrew" | "greek";
+}
```

Comment:
For this first version, I'd like to simplify this to a single button for the user. Let's use your standard layout and export all books by default even if there are no glosses for that book yet. This is the MVP of this feature and we can expand options later.